src/main/resources/org/jenkinsci/plugins/codesonar/CodeSonarPublisher/help-visibilityFilterNewWarnings.html

```diff
@@ -0,0 +1,15 @@
+<div>
+    <p dir="auto">A CodeSonar warning visibility filter that will be used by the CodeSonar plugin to determine the list <i>new</i> of warnings when evaluating analysis condition options.</p>
```

Comment:
A CodeSonar warning visibility filter.  **CodeSonar condition** based on "number of new warnings" are evaluated with respect to this filter.